keyboards/mixi/keymaps/via/keymap.c

```diff
@@ -58,7 +59,7 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 };
```

Comment:
```suggestion
    [3] =
        LAYOUT(
                KC_NO, KC_NO, KC_NO, 
                KC_NO, KC_NO, KC_NO, 
                KC_NO, KC_NO, KC_NO
              )
};
```
VIA uses 4 layers by default. I don't see this value overridden for this board, so I recommend a 4th, emtpy layer.

**Note**: Because of this, Line57 will need a semicolon.

Reply:
Applied at new commit.